E621Client/Area/Tag/E621Client.cs

```diff
@@ -151,28 +64,7 @@ private Task<ICollection<Tag>> GetTagsByNamesAsync(IEnumerable<string> names, in
 
         #region Tags filter with query
 
-        /// <summary>
-        /// <para>Retrieves a collection of tags using a query and the other given parameters.</para>
-        /// <para>
-        /// Be aware that there is a limit on the number of tags which can be retrieved with a
-        /// single call. This limit is defined at <see cref="TagsMaximumLimit"/>. Exceeding this
-        /// limit, will cause an exception to be thrown.
-        /// </para>
-        /// <para>
-        /// You can navigate through all the tags by specifying a page number. There is, however, a
-        /// limit to the value. This limit is defined in <see cref="TagsMaximumPage"/>. Exceeding
-        /// this limit, will cause an exception to be thrown.
-        /// </para>
-        /// </summary>
-        /// <param name="query">The query used to filter the collection with.</param>
-        /// <param name="page">Pagination, number of the page.</param>
-        /// <param name="limit">The maximum number of tags returned in a single request.</param>
-        /// <param name="category">The category the retrieved tags should be in.</param>
-        /// <param name="order">The order in which the retrieved tags should be.</param>
-        /// <param name="hideEmpty">
-        /// Whether or not to include tags that have no posts associated with them.
-        /// </param>
-        /// <param name="hasWiki">Whether or not to include tags that have a page at e621's wiki.</param>
+        /// <inheritdoc cref="IE621Client.GetTagsByNames(System.Collections.Generic.IEnumerable{string},System.Nullable{int},System.Nullable{int},System.Nullable{Noppes.E621.TagCategory},System.Nullable{Noppes.E621.TagOrder},bool,System.Nullable{bool})"/>
```

Comment:
Wrong `cref` again.

Reply:
I had some trouble changing this at first but now i've figured it out.